docs/docs/getting-started/first-steps.md

```diff
@@ -39,182 +39,48 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
 2. Fill in your cloud provider specific information.
 
+    First you need to create an [IAM configuration](../workflows/config.md#creating-an-iam-configuration). The easiest way to do this is the following CLI command:
+
     <tabs groupId="csp">
-    <tabItem value="azure" label="Azure (CLI)">
 
-    You need several resources for the cluster. You can use the following `az` script to create them:
```

Comment:
There's a tip at the end of this page regarding this az script, which doesn't make sense anymore and should be removed.